verification/simulator.go

```diff
@@ -27,7 +27,7 @@ const (
 	DPE_SIMULATOR_VENDOR_SKU            uint32  = 0
 )
 
-type DpeSimulator struct {
```

Comment:
I think we should leave this as DpeSimulator. The DpeEmulator should just also implement the TestDPEInstance interface (but the implementation will be different)